chill-protobuf/src/main/java/com/twitter/chill/protobuf/ProtobufSerializer.java

```diff
@@ -43,21 +43,30 @@
    * classes in play, which should not be very large.
    * We can replace with a LRU if we start to see any issues.
    */
-  final protected HashMap<Class, Method> methodCache = new HashMap<Class, Method>();
+  final protected HashMap<Class, Method> parseMethodCache = new HashMap<Class, Method>();
```

Comment:
can we keep the name `methodCache` to not change binary compatibility? We can just comment that this is the parseMethod?

Reply:
@johnynek Good point. Done!